python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -195,9 +201,7 @@ def __from_edgelist(
             )
             source = renumber_map.renumbered_src_col_name
             destination = renumber_map.renumbered_dst_col_name
-            # Use renumber_map to figure out if renumbering was skipped or not
-            # This was added to handle 'legacy_renum_only' which will skip the
-            # old C++ renumbering  when running the pylibcugraph/C algos
+            # Use renumber_map to figure out if the python renumbering occured
             self.properties.renumbered = renumber_map.implementation.numbered
```

Comment:
I seem to be getting below error  on this, can you recheck

```
stributedGraphImpl.__from_edgelist(self, input_ddf, source, destination, edge_attr, renumber, store_transposed, legacy_renum_only)
    232 # If renumbering is not enabled, this function will only create
    233 # the edgelist_df and not do any renumbering.
    234 # C++ renumbering is enabled by default for algorithms that
    235 # support it (but only called if renumbering is on)
    236 self.compute_renumber_edge_list(
    237     transposed=store_transposed, legacy_renum_only=legacy_renum_only
    238 )
--> 240 self.properties.renumbered = self.renumber_map.implementation.numbered
    241 ddf = self.edgelist.edgelist_df
    243 num_edges = len(ddf)

AttributeError: 'NoneType' object has no attribute 'implementation'
```

Reply:
@jnke2016 it might be better to add a proper method or bool property to NumberMap, like `is_renumbered` instead of trying to access the internal objects.

Reply:
I believe I resolved this issue per @rlratzel recommendation but feel free to test it again and let me know @VibhuJawa

Reply:
I think your tests should catch if the error would have happened.